### PR TITLE
raise contrast on a few key page elements to pass WCAG contrast checks

### PR DESCRIPTION
**Pull request by ryanpetrello**
Several page elements and labels were failing the contrast check for me.  This raises the contrast slightly to pass WCAG.

### CODE REVIEW

**Comment by ryanpetrello** [comment]
kapture 2018-11-27 at 9 42 02

**Comment by ryanpetrello** [comment]
I clicked all over the UI, and I only see two remaining offenders.  First, it doesn't like the green status circles because it thinks there's text in them (you'll see that example in my GIF above).  Second, it doesn't like contrast of these line numbers for code blocks, but I feel somewhat meh about it given the line numbers aren't really that useful/meaningful anyways:

image

File: awx/ui/client/features/output/_index.less [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'host-event/_index';
 .at-Stdout {
     &-menuTop {
-        color: @at-gray-848992;
+        color: @at-gray-646972;
```
Comment (ryanpetrello) [inline awx/ui/client/features/output/_index.less:4]: This was too light for labels throughout the UI.

File: awx/ui/client/features/output/constants.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export const JOB_STATUS_FINISHED = JOB_STATUS_COMPLETE.concat(JOB_STATUS_INCOMPL
 export const OUTPUT_ANSI_COLORMAP = {
     0: '#000',
     1: '#A00',
-    2: '#0A0',
+    2: '#080',
```
Comment (ryanpetrello) [inline awx/ui/client/features/output/constants.js:19]: The default ANSI green color we were using was too light, I've darkened it slightly.

image

File: awx/ui/client/legacy/styles/codemirror.less [hunk 3]
```diff
@@ -31,8 +31,8 @@
     background-color: @default-no-items-bord;
 }
 
-.CodeMirror-lineNumber {
-    color: @default-icon;
+.CodeMirror-linenumber {
+    color: @at-gray-70;
```
Comment (ryanpetrello) [inline awx/ui/client/legacy/styles/codemirror.less:35]: image

File: awx/ui/client/src/shared/branding/colors.default.less [hunk 4]
```diff
@@ -1,7 +1,7 @@
 // default base colors
-@default-interface-txt: #707070;
+@default-interface-txt: #606060;
```
Comment (ryanpetrello) [inline awx/ui/client/src/shared/branding/colors.default.less:2]: Interface text throughout was slightly too light in various places, this minor adjustment makes us pass everything I could find.

File: awx/ui/client/features/output/_index.less (hunk 1 above)
Comment (ryanpetrello) [reply awx/ui/client/features/output/_index.less:4]: image

vs

image

File: awx/ui/client/legacy/styles/codemirror.less (hunk 3 above)
Comment (ryanpetrello) [reply awx/ui/client/legacy/styles/codemirror.less:35]: `#707070` was the sweet spot here.

**Review by trahman73 — APPROVED** [review]
Looks good!

**Pull request merged**